### PR TITLE
Add optional chaining to author in teaser posts list

### PR DESCRIPTION
**Pull request by santiagoalex**
Added optional chaining to the author in teaser posts list to prevent fall application because it is possible to posts that don't have an author

**What problem is this solving?**
There is a breakdown in the application when listing category teasers and this does have the author in posts

Error_teaser_no_existe_author

**How should this be manually tested?**

Install this branch in your workspace and use a WordPress that doesn´t  have authors in posts

**Screenshots or example usage:**

Install this versión in your workspace by typing `vtex install` in your root project

### CODE REVIEW

**Comment by santiagoalex** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! robot
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually disappointed

**Comment by santiagoalex** [comment]
> Beep boop robot
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document thinking
> * [ ]  I'll do it later disappointed
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated tadatada

**Comment by santiagoalex** [comment]
Good morning @Syed-Vtex, I hope you are okay.

it would help a lot if this PR could be reviewed, it would prevent render issues from the app on the categories  page

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Add option to link to blog post AMP pages on external WordPress site when AMP plugin used
+## [2.19.3] - 2022-05-30
```
Comment (polishq) [inline CHANGELOG.md:14]: Please remove this `2.19.3` entry and move your `Fixed` item up into the `Unreleased` section at the top of the changelog. It should look like this:

```
## [Unreleased]

### Fixed

- Add conditional ...

## [2.20.0] - 2022-06-22

### Added
...
```

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wordpress-integration",
   "vendor": "vtex",
-  "version": "2.20.0",
+  "version": "2.20.1",
```
Comment (polishq) [inline manifest.json:4]: Do not change the version number; a GitHub bot will do this automatically when the PR is merged

**Review by polishq — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (santiagoalex) [reply CHANGELOG.md]: Thanks a lot for your feedback!

File: react/components/WordpressCategory.tsx [hunk 3]
```diff
@@ -331,4 +331,4 @@ WordpressCategory.schema = {
   },
 }
 
-export default WordpressCategory
+export default WordpressCategory
```
Comment (polishq) [inline react/components/WordpressCategory.tsx:334]: Add a new line at the end of this file to fix the lint error

**Review by polishq — CHANGES_REQUESTED** [review]

**Pull request merged**